src/sota_tools/deploy.cc

```diff
@@ -94,18 +94,19 @@ bool copy_repo(const std::string &cacerts, const std::string &src, const std::st
 
   if (!src_repo->LooksValid()) {
     LOG_FATAL << "The OSTree src repo does not appear to contain a valid OSTree repository";
-    return EXIT_FAILURE;
+    return false;
   }
 
   OSTreeRef ostree_ref(src_repo->GetRef(ref));
   if (!ostree_ref.IsValid()) {
     LOG_FATAL << "Ref " << ref << " was not found in repository " << src;
```

Comment:
`src` is just the name of the zipped credentials file; this should probably print the repo name or location instead.

Reply:
No src it is folder or url.

Reply:
I just veryfied and everything works for me. Are You sure You pass the exists refname?

Reply:
It does work for me, but @Danten was trying to put this into an end-to-end test and saw the above problem. I checked with him that everything was where we expected it to be.